COPYRIGHT.txt

```diff
@@ -1135,7 +1135,7 @@ License: glslang
  GNU General Public License for more details.
  .
  You should have received a copy of the GNU General Public License
- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
this seem to have been updated to https in the version 3, so I think this is fine?

Reply:
No this was wrong. This is the glslang license, and it uses http. We should *not* modify thirdparty project license terms.

You can PR the change to https://github.com/KhronosGroup/glslang/blob/master/LICENSE.txt#L321 but until approved upstream, we shouldn't change their license.